changelog/traits.dd

```diff
@@ -0,0 +1,11 @@
+Enable private member access for traits
+
+The following traits can now access non-public members:
+$(UL
+    $(LI getMember)
+    $(LI getOverloads)
+)
+
+This fixes a long-standing issue in D where the allMembers trait would correctly return non-public members but those non-public members would be inaccessible to other traits.
```

Comment:
The problem I and other encountered is also tied to the fact that a library trait located in one module could not work at a at the use site because it was instantiated elsewhere. The same library trait, copied (or mixed in) at the use site worked just fine.

Reply:
I forgot to say but somewhere it should be documented that from now it's a good practice to check the protection and to apply it unless "you know what you do".